conference/tests/templatetags/test_conference.py

```diff
@@ -0,0 +1,46 @@
+import mock
+from django.template import Context
+from django.test import RequestFactory
+from django.test import TestCase
+
+from conference.tests.factories.conference import ConferenceFactory
+from conference.tests.factories.talk import TalkFactory
+
+
+class PrivateFunctionsTestCase(TestCase):
```

Comment:
maybe instead of classes we can use just functions, since we don't have a setup method